### ISSUE TITLE
Plugin not found

### ISSUE THREAD

**Issue by Dorian** [heroku/heroku-pg-extras#139]
```
heroku plugins:install heroku-pg-extras -a my-app
Installing plugins heroku-pg-extras -a my-app... !
 ▸    Plugin not found
```

**Comment by uhoh-itsmaciek** [comment]
@Dorian interesting, this seems to be an issue with the `-a` flag handling:

```
maciek@mothra:~$ heroku plugins:install heroku-pg-extras
Installing plugin heroku-pg-extras... done
maciek@mothra:~$ heroku plugins:install heroku-pg-extras -a shogun-maciek
Installing plugins heroku-pg-extras -a shogun-maciek... !
 ▸    Plugin not found
```

Can you try it again without `-a my-app` and see if that works for you?

/cc @dickeyxxx

**Comment by Dorian** [comment]
@uhoh-itsmaciek same with `--app` BTW.

So as you said with the app it works:

```
heroku plugins:install heroku-pg-extras
Installing plugin heroku-pg-extras... done
```

For that, I had to add the `heroku` remote to my `.git/config`, which I don't like, anyway here it is fro anyone wondering:

```
[remote "heroku"]
  url = git@heroku.com:my-app.git
  fetch = +refs/heads/*:refs/remotes/heroku/*
```

**Comment by dickeyxxx** [comment]
Doing that is attempting to install 3 plugins. heroku-pg-extras, -a and the
app name. That's why it fails

**Comment by uhoh-itsmaciek** [comment]
@Dorian you had to add the heroku remote to your `.git/config`? But `plugins:install` doesn't require an app at all--what happens when you run the bare `heroku plugins:install heroku-pg-extras` without the app flag?

@dickeyxxx ahhhh! Thanks, that makes sense! Can we guard against that, or note which plugins are failing to install? So many commands do require an app that it's easy to accidentally add it here.

**Comment by Dorian** [comment]
@uhoh-itsmaciek Oh! Nice, thanks, I didn't realize it was just a local program and not associated with an app.

**Comment by dickeyxxx** [comment]
Possibly, we don't really need the multi-plugin install thing anymore so we could potentially get rid of it

**Comment by jpwynn** [comment]
How does one specify an app, if the -a on the CLI is now mistaken for another plugin. Or alternately, if you have multiple heroku apps in git/config how does one specify _which_ remote heroku app to use? 

(we have one for staging that we push to directly, one for production that we use heroku pipelines:promote)

**Comment by Dorian** [comment]
@jpwynn it's a local plugin so it's gonna make request to the database of your app when you use it. But it's not gonna change anything on your dynos

**Comment by jpwynn** [comment]
Yes I know. My question is, once installed, how do you specify WHICH app for the plugin to use? Your git can have several remotes for example a staging app and a production app. This USED to work but no longer does:

heroku pg:cache-hit -r my_git_remote_for_app
or
heroku  pg:cache-hit -a my_heroku_appname

because suddenly now it seems anything on the command line is assumed to be a plugin, eg, no provision for command line args?

**Comment by dickeyxxx** [comment]
@jpwynn what error are you seeing exactly?

**Comment by jpwynn** [comment]
It is working now... as of heroku-pg-extras@1.0.8

I removed the old version plus the new version explicitly, then reinstalled it so it might have been a conflict arising from the old version being left behind when the new version was installed.
